src/app.js

```diff
@@ -411,7 +411,6 @@ var run = function () {
     setEditorSize(hidingRHP ? 0 : storage.getEditorSize());
     $('.toggleRHP i').toggleClass('fa-angle-double-right', !hidingRHP);
     $('.toggleRHP i').toggleClass('fa-angle-double-left', hidingRHP);
-    if (!hidingRHP) compiler.compile();
```

Comment:
Before contracts were only rendered if (hidingRHP ==  false), now contracts are rendered in each case (even if the panel is hidden) so we don't need to recompile.
I know that it could look like a waste of resources to recompile even if the panel is hidden but:
 - In most of the case, the panel will not be hidden cause compilation errors are displayed in this panel.
 - Basicallly i wanted to remove getHidingRHP as a dependency of renderer.js. It looked like an unnecessary dependency.